geometries/eclipse/eclipse_algorithm.py

```diff
@@ -18,7 +18,7 @@ def eclipse(utc, eclipsed_body,target):
     # Front body
     front = 'EARTH'
     fshape = 'ELLIPSOID'
-    fframe = 'ITRF93'
+    fframe = f'IAU_{front}'
```

Comment:
There is no need to use ITRF93 at this point as IAU_EARTH is good enough for testing.  IRTF93 was removed from the test kernel set to save space.

ITRF93 may be added back here later, or perhaps a keyword added to the call to choose which to use.